.github/workflows/functional-test.yaml

```diff
@@ -380,7 +386,9 @@ jobs:
       - name: Login to Azure
         uses: azure/login@v1
         with:
-          creds: '{"clientId":"${{ secrets.INTEGRATION_TEST_SP_APP_ID }}","clientSecret":"${{ secrets.INTEGRATION_TEST_SP_PASSWORD }}","subscriptionId":"${{ secrets.INTEGRATION_TEST_SUBSCRIPTION_ID }}","tenantId":"${{ secrets.INTEGRATION_TEST_TENANT_ID }}"}'
+          client-id: ${{ secrets.INTEGRATION_TEST_SP_APP_ID }}
```

Comment:
While we're in here making changes can we switch this over to secrets.AZURE_SP_TESTS_APPID?

It's the same service principal behind the scenes, but we've been using separate secrets for it. A couple years ago we switched to an org-scoped secret but this workflow wasn't updated to use it.

Reply:
![image](https://github.com/radius-project/radius/assets/54363786/1cb1a5a1-49c1-4413-bef1-9dd299ad9d0d)

Reply:
Updated. Do you have new subscription id for test ?

Reply:
Ah yeah `AZURE_SUBSCRIPTIONID_TESTS` should be the correct value:
![image](https://github.com/radius-project/radius/assets/54363786/87ccaf58-ff3b-4e9b-b8d1-3a0160925aeb)

Although, would a variable be better than a secret since it's not a sensitive value?

Reply:
> Ah yeah `AZURE_SUBSCRIPTIONID_TESTS` should be the correct value: ![image](https://private-user-images.githubusercontent.com/54363786/313771377-87ccaf58-ff3b-4e9b-b8d1-3a0160925aeb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA3ODIzNzEsIm5iZiI6MTcxMDc4MjA3MSwicGF0aCI6Ii81NDM2Mzc4Ni8zMTM3NzEzNzctODdjY2FmNTgtZmYzYi00ZTliLWI4ZDEtM2EwMTYwOTI1YWViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE4VDE3MTQzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk0NGMyNjFlMTBlODI2NWIzMjY4ZmU4NDIyMTI1MzIzYzdjZDdhMWRkYmQxNDBhNmFiMjZhZDMyODhlY2IxMTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.g3iKRWeUh-K7gfg9fjbOhEpMLXcOznoPiAyE73vCoxE)
> 
> Although, would a variable be better than a secret since it's not a sensitive value?

secret is better than variable. if you can create new secret, please let me know.

Reply:
If it is being added as a secret, then can you make sure to document the value for AZURE_SUBSCRIPTIONID_TESTS in our wiki?

Reply:
> If it is being added as a secret, then can you make sure to document the value for AZURE_SUBSCRIPTIONID_TESTS in our wiki?

looks like Aaron added it already.